FreeRTOS/FreeRTOS
# Removing FreeRTOS-Labs directory.

Description
-----------
Removing ./FreeRTOS-Labs directory, since:
    - IoT libraries are now in LTS branch.
    - FAT/POSIX/Light-weight MQTT are in https://github.com/FreeRTOS/FreeRTOS-Labs.

As a side effect, FAT SL demo source code is also removed. This is expected, as FAT SL product was removed some time ago.

Test Steps
-----------
N/A

We'll need to clean up web documentations for FAT SL as follow up action item. e.g. https://www.freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_FAT_SL/FreeRTOS_Plus_FAT_SL.html. 

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
